rfcs/react-components/non-core-components.md

```diff
@@ -0,0 +1,388 @@
+- 👎 Need to handle urgent bugfixes for most past versions - need a clear support SLA
+
+### Choosing a repo for
+
+#### Current monorepo
+
+Non-core packages should still live in the monorepo.
+
+##### Pros
+
+- 👍 Easier setup
+- 👍 Nothing new to learn
+- 👍 Issue reporting path does not change
+
+##### Cons
+
+- 👎 We might not want the same release cadence - releasing a fix for a non-core component would release all of core
+- 👎 Extra load on the CI as more non-core components are needed
+- 👎 Not possible to have more separation from core components without lots of tooling hacks
+
+#### Separate monorepo
+
+We can create a new monorepo for our non-core packages and depend on core packages from NPM.
+
+##### Pros
+
+- 👍 We can have our ideal DX
+- 👍 Release cadence does not need to depend on core
+- 👍 Developing our with our public API - we eat our own dogfood
+
+##### Cons
+
+- 👎 Needs effort to setup
+- 👎 Issue reporting becomes more unclear
+- 👎 Requires linking to use latest core code
+- 👎 Manual/Automated bumps required to keep packages up to date
+
+### Package scope
+
+#### @fluentui scope
+
+Non-core packages would be published under the same `@fluentui/` NPM scope.
+
+##### Pros
+
+- 👍 No need to create and manage another organization
+- 👍 Easier discoverability for OSS users and internal partners
+
+##### Cons
+
+- 👎 Not clear that there is a difference between non-core components and core components/utilities
+- 👎 Harder for us to tell the difference between non-core and core
+
+#### New package scope
+
+Creates a new package scope, `@fluentui-banana/` (name pending) for all components/packages that are non-core.
+
+##### Pros
+
+- 👍 Very clear differentiation between core and non-core
+- 👍 Creates a clear boundary for FUI team and partners to collaborate
+- 👍 Clear path to upgrade/downgrade packages between core and non-core (i.e. experiments/compat)
+
+##### Cons
+
+- 👎 Need to maintain a new NPM organization and credentials
+- 👎 Initially could cause some confusion if we don't communicate this well
+
+### How users should consume components
+
+We should keep the same architecture we currently have for our core components: one package per component. What should
+be decided here is how users consume these components.
+
+#### Suite packages
+
+Structure non-core components into a new suite package similar to `@fluentui/react-components`.
+
+##### Pros
+
+- 👍 Similar to core components
+- 👍 Easy to update when using multiple components
+
+##### Cons
+
+- 👎 Needs to follow the breaking change cadence of each components
+- 👎 Slows down breaking changes for all components
+
+#### Individual packages
+
+Users should install each component package individually.
+
+##### Pros
+
+- 👍 Easy to pick and choose components that might not always be relevant
+- 👍 Allows more flexible breaking change cadence
+
+##### Cons
+
+- 👎 Easier to install duplicate versions
+- 👎 We might be encouraged to break too often
+
+### How to depend on core components
```

Comment:
both approaches are valid here if we can maintain/enforce to use `^` on core etc. 

peerDeps seems to be a better and more used solution in general but from experience they add another set of problems. (as you mentioned the major bump situation, npm enforcement while installing etc )

we can start with peerDeps and switch to deps as needed